### PR TITLE
Fixing typos in uninstall tool documentation

### PR DESCRIPTION
**Pull request by sfoslund**
## Summary

Fixing typos in uninstall tool documentation: 
- The description (remove all x86 .NET Core runtimes except…) is in-consistent with example command (remove all runtimes except…). 
- There is a typo in the example of the tool. It should show"--verbosity" rather than "-verbosity"

### CODE REVIEW

File: docs/core/additional-tools/uninstall-tool.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Uninstall Tool
 description: An overview of the .NET Core Uninstall Tool, a guided tool that enables the controlled clean-up of .NET Core SDKs and runtimes.
 author: sfoslund
-ms.date: 12/17/2019
+ms.date: 1/6/2019
```
Comment (Youssef1313) [inline docs/core/additional-tools/uninstall-tool.md]:
```suggestion
ms.date: 1/6/2020
```

File: docs/core/additional-tools/uninstall-tool.md [hunk 2]
```diff
@@ -464,7 +464,7 @@ Notes:
 > [!NOTE]
 > By default, .NET Core SDKs and runtimes that may be required by Visual Studio or other SDKs are kept. In the following examples, some of the specified SDKs and runtimes may remain, depending on the state of the machine. To remove all SDKs and runtimes, list them explicitly as arguments or use the `--force` option.
 
-* Remove all x86 .NET Core runtimes except the version `3.0.0-preview6-27804-01` without requiring Y/n confirmation:
+* Remove all .NET Core runtimes except the version `3.0.0-preview6-27804-01` without requiring Y/n confirmation:
```
Comment (mairaw) [inline docs/core/additional-tools/uninstall-tool.md]: How does the confirmation works? Wondering if we could say here yes/no confirmation.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @sfoslund. The changes look good. I have a question for you re. an existing text + you should apply @Youssef1313's suggestion.

File: docs/core/additional-tools/uninstall-tool.md (hunk 2 above)
Comment (sfoslund) [reply docs/core/additional-tools/uninstall-tool.md]: The user is asked to enter `Y` for confirmation, and right now anything besides that (including `yes`) is taken as no. I plan on changing this but for right now that's the situation.

Comment (mairaw) [reply docs/core/additional-tools/uninstall-tool.md]: Ok. Maybe we should just capitalize like this then?

```suggestion
* Remove all .NET Core runtimes except the version `3.0.0-preview6-27804-01` without requiring Y/N confirmation:
```

**Review by mairaw — APPROVED** [review]

**Pull request merged**